src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+
+    // we should be OK here to have it suppressed
+    // since Pair instances are not shared with outside world
+    @SuppressWarnings("unchecked")
+    private final Pair<K, V>[] pairs = new Pair[MAX_SIZE];
```

Comment:
For the future, it is better to use constructor for such filed
```suggestion
    private final Pair<K, V>[] pairs;
```